manifests/a/Amazon/AWSCLI/2.2.35.0/Amazon.AWSCLI.locale.en-US.yaml

```diff
@@ -1,12 +1,14 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
 PackageIdentifier: Amazon.AWSCLI
 PackageVersion: 2.2.35.0
 PackageLocale: en-US
 Publisher: Amazon Web Services
 PublisherUrl: https://github.com/aws/aws-cli
 PublisherSupportUrl: https://github.com/aws/aws-cli/issues
 Author: Amazon Web Services
-PackageName: AWS Command Line Interface
+PackageName: AWS Command Line Interface v2
```

Comment:
Before we do this, it's probably worthwhile to wait until all of the `Amazon.AWSCLI` (v2) packages are moved to `Amazon.AWSCLI.2` and also have their Publisher and PackageName keys updated so that we can create `Amazon.AWSCLI.1` as v1 still receive updates.